### DISCUSSION TITLE
[Question]: What powered USB hubs are you all using for Coral?

### DISCUSSION THREAD

**Discussion by rocket59** [blakeblackshear/frigate#10071] [Ask A Question]
### What is your question:

I get the occasional "detection stuck" error on my setup (dell 5070/proxmox LXC) which I'm guessing "might" be due to excess power draw in certain high motion situations. I don't think it's the cable as the error is very intermittent.

If you have a working USB hub setup, would you be able to share the brand and model?

**Comment by allaboutmikey** [comment]
I have a TP-Link TL-SG1210MP
Which has 8 POE ports and has been working nicely for just over a month now.
